src/verifiers/issuerIdentity/index.ts

```diff
@@ -0,0 +1,27 @@
+import { issuerIdentityVerifierBuilder } from "./builder";
```

Comment:
Overall,

I find this version much more harder to use than the previous implementation:

- I cant upgrade opencerts to use this version without writing my full own identity implementation (I can't have multiple verifier for one issuer) + validity
- There are different logic for different kind of fragments (I cant mix document_status fragment but I can mix issuer_identity fragment)
- It's not more composable than before (personal opinion but prove me wrong)
- I would not upgrade to this version for opencerts and would rather fix isValid according to my need

Reply:
The problem with this actually go away in V3, the problem was that our method for verifying multiple identities in V2 is incorrect (not inelegant). 

When you run multiple verifiers against a list of identities it is impossible to ensure the validity of the issuers if they use different schemes. 

Ie. 
Issuer 1 - DNS-TXT
Issuer 2 - Opencerts Registry
Issuer 3 - Non-existent method

Having both Opencerts registry and Dns-txt will skip past the last issuer and return VALID where it should have been INVALID when *not all* the issuers are verified. 

This problem goes away in V3 when there is only one issuer and that we can run all the independent fragments like before... I'll describe the upgrade process for OpenCerts registry in the comment below.

Reply:
Conversation Continued:

Rui (Govtech), [27.10.20 15:29]
probably better to do the discussion on the github thread because if not need to copypaste to laurent again since the comments came from him

Rui (Govtech), [27.10.20 15:29]
there's an issue with correctness

Rui (Govtech), [27.10.20 15:30]
because way back in the day we made that original sin of deciding that oneOf(identity) is sufficient to be valid

Rui (Govtech), [27.10.20 15:30]
actually that's not correct either

Raymond Yeh, [27.10.20 15:30]
[In reply to Rui (Govtech)]
ehh we didnt decide on this

Rui (Govtech), [27.10.20 15:30]
its more like

if (registry || oneOf(other identities))

Raymond Yeh, [27.10.20 15:30]
it was suppose to be allOf but i think because we didnt think there were more than one way to verify identity

Rui (Govtech), [27.10.20 15:31]
nah we made that decision i remember we had that conversation

Raymond Yeh, [27.10.20 15:31]
if (registry || oneOf(other identities))
^ for single issuer you see....

Raymond Yeh, [27.10.20 15:31]
so should be 
allOf ( (registry || oneOf(other identities)) )

Raymond Yeh, [27.10.20 15:32]
we had not been thinking about the case about (multiple issuer) x (different way to verify)

Rui (Govtech), [27.10.20 15:32]
:D.........

Raymond Yeh, [27.10.20 15:33]
[In reply to Rui (Govtech)]
also, this is reduced to 
(registry || verifier 1 || verifier 2 || etc) 
^ which is the premise of the identity builder